examples/projects/NUCLEO-L073RZ/bootloader/include/gpio.h

```diff
@@ -0,0 +1,49 @@
+/* USER CODE BEGIN Header */
```

Comment:
I think your commit comment should be `Add bootloader and 2  gate and button exemples` instead of `add bootloader and 2 exemple gate and button`